### PR TITLE
chore: add K8s sept hotfix and 8b image

### PR DESCRIPTION
**Pull request by mainred**
**Reason for Change**:

- Add k8s v1.16.13 v1.16.15 v1.17.9 v1.17.11 v1.18.6 hotfix and v1.18.8 v1.19.0
- Upgrade windows image version which have the above winzip cached

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mainred** [comment]
@AbelHu

**Review by AbelHu — CHANGES_REQUESTED** [review]

**Review by AbelHu — APPROVED** [review]
Suggest to run `make dev; make generate` to assure template_generated.go is the latest.

**Comment by mainred** [comment]
@AbelHu , thanks for your suggestion. The last change from `make generate` is mainly for formating the code, containing no actual code change. Still I pushed this change because other people may benefit from this.

**Comment by mainred** [comment]
@xuto2 @bowang-666 this includes the hotfix and windows new VHD to be included in next Monday.

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -184,9 +184,9 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the AKS image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2008",
+		ImageSku:       "aks-2019-datacenter-core-smalldisk-2009",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.1397.200820",
+		ImageVersion:   "17763.1397.200904",
```
Comment (bowang-666) [inline pkg/api/azenvtypes.go:189]: This file has been moved to RP, you can just update there

Comment (mainred) [reply pkg/api/azenvtypes.go:189]: Thanks @bowang-666 , filed a PR in  aks-rp, will close this PR.

File: pkg/api/k8s_versions.go [hunk 2]
```diff
@@ -182,16 +188,22 @@ func getKubeConfigs() map[string]map[string]string {
 
 func getVersionOverrides(v string) map[string]string {
 	switch v {
+	case "1.18.6":
```
Comment (AbelHu) [inline pkg/api/k8s_versions.go:191]: Updated pkg\api\common\versions.go to set these supported versions to true.

Comment (mainred) [reply pkg/api/k8s_versions.go:191]: have a full sync from aks-rp https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/3469550?_a=files&path=%2Fcommon%2Fk8sversions%2Fversions.go

Comment (mainred) [reply pkg/api/k8s_versions.go:191]: done, thanks.

**Review by AbelHu — APPROVED** [review]

**Pull request merged**